routes/beetles.py

```diff
@@ -0,0 +1,578 @@
+import asyncio
+import io
+import csv
+import calendar
+import numpy as np
+from scipy import stats as st
+from math import floor
+from flask import Blueprint, render_template, request, Response
+from shapely.geometry import Point
+
+# local imports
+from generate_urls import generate_wcs_query_url
+from generate_requests import *
+from fetch_data import *
+from fetch_data import (
+    fetch_data,
+    get_from_dict,
+    summarize_within_poly,
+    csv_metadata,
+    fetch_wcs_point_data,
+)
+from validate_request import (
+    validate_latlon,
+    project_latlon,
+    validate_var_id,
+    validate_year,
+)
+from validate_data import (
+    get_poly_3338_bbox,
+    nullify_and_prune,
+    postprocess,
+    place_name_and_type,
+)
+from luts import type_di
+from . import routes
+from config import WEST_BBOX, EAST_BBOX
+
+beetles_api = Blueprint("beetles_api", __name__)
+# Rasdaman targets
+beetle_coverage_id = "beetle_risk"
+#
+# beetle_dim_encodings = asyncio.run(
+#     get_dim_encodings("template_beetle_risk")
+# )
```

Comment:
These encodings are now on Rasdaman (Apollo) so you might be able to remove the `dim_encodings` block and use this commented out code instead. Dealer's choice between that and just removing the commented-out code.